cmd/cloud/deployment.go

```diff
@@ -360,3 +371,50 @@ func deploymentVariableUpdate(cmd *cobra.Command, args []string, out io.Writer)
 
 	return deployment.VariableModify(deploymentID, variableKey, variableValue, ws, envFile, variableList, useEnvFile, makeSecret, true, astroClient, out)
 }
+
+func newDeploymentWorkerQueueRootCmd(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "worker-queue",
+		Aliases: []string{"wq"},
+		Short:   "Manage deployment worker queues",
+		Long:    "Manage worker queues for an Astro Deployment.",
+	}
+	cmd.AddCommand(
+		newDeploymentWorkerQueueCreateCmd(out),
+	)
+	return cmd
+}
+
+func newDeploymentWorkerQueueCreateCmd(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "create",
+		Aliases: []string{"cr"},
+		Short:   "Create a Deployment's worker queue",
+		Long:    "Create a worker queue for an Astro Deployment",
+		Example: "", // TODO should we add some examples for default and custom worker-queue commands?
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return deploymentWorkerQueueCreate(cmd, args, out)
+		},
+	}
+	cmd.Flags().StringVarP(&deploymentID, "deployment-id", "d", "", "The deployment where the worker queue should be created.")
+	cmd.Flags().StringVarP(&wQueueName, "name", "n", "", "The name of the worker queue. If the name contains a space, specify the entire name within quotes \"\" .")
+	cmd.Flags().BoolVarP(&isDefaultWQueue, "isDefault", "", false, "Make this the default worker queue by setting this to true.")
```

Comment:
If we expose this, does this mean under the hood we set the original default queue's `isDefault` field to false? I don't think this flag is intuitive for the user. Additionally, the default queue has naming constraints.

Reply:
@frank-ye does that mean that a user should not be able to create a new queue and make it the "default" queue? In other words we should not have this flag at all? 

@sunkickr thoughts on why we need this flag?

Reply:
@frank-ye I spoke to @sunkickr and understand the use case better so have removed the `--isDefault` flag as users can not create a default worker queue.